### DISCUSSION TITLE
Can we implement our own rules?

### DISCUSSION THREAD

**Discussion by sethamus** [rome/tools#4710] [Q&A] [answered]
I am having a bit of hard time understanding this project. Can we use eslint plugins, like unicorn or import, with rome? If not then is there a way to create your own rules? Or are we stuck with what is available with rome?

**Comment by ematipico** [comment, marked as answer]
There isn't a way to create your rules yet.

**Reply by sethamus** [reply]
@ematipico Do you have plans on adding such feature or is it out of scope for this project?

**Reply by ematipico** [reply]
We do, but it's something we're going to add later. I don't have a timeline unfortunately.
